src/Microsoft.DotNet.Cli.Utils/HashHelper.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class HashHelper
+    {
+        public static string sha256_hash(string value)
+        {
+            StringBuilder Sb = new StringBuilder();
```

Comment:
If this method took a string[], we could hash all the args with 1 stringBuilder and 1 SHA256 instance being created. Instead of needing to create new instances for each arg.